### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by sbellem**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
-    - ./setup.cfg:/usr/src/app/setup.cfg
-    - ./setup.py:/usr/src/app/setup.py
-  command: py.test --verbose
+version: '2'
+
+services:
+  spool:
+    build: .
+    volumes:
+      - ./docs:/usr/src/app/docs
```
Comment (sohkai) [inline docker-compose.yml:7]: Is this mapping necessary?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+      - ./tests:/usr/src/app/tests
+      - ./pytest.ini:/usr/src/app/pytest.ini
+      - ./setup.cfg:/usr/src/app/setup.cfg
+      - ./setup.py:/usr/src/app/setup.py
+    command: py.test --verbose
+  bdocs:
+    build: .
+    volumes:
+      - ./docs:/usr/src/app/docs
+      - ./spool:/usr/src/app/spool
```
Comment (sohkai) [inline docker-compose.yml:18]: Is this mapping necessary?

File: docs/source/api.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+=========================
+
+Spool
+-----
+.. automodule:: spool
+
+.. autoclass:: Spool
+    :members:
+
+.. :members: __init__, register_piece, register, editions, consigned_registration, transfer, consign, unconsign, loan, migrate, refill_main_wallet, refill, simple_spool_transaction, select_inputs
```
Comment (sohkai) [inline docs/source/api.rst:11]: I assume this is where the current [documentation section](https://github.com/ascribe/pyspool#documentation) in the README will go?

**Comment by sohkai** [comment]
If we'll put it up on readthedocs soon, we might as well remove the duplicate sections in the README as well. Otherwise looks good 👍.

File: docker-compose.yml (hunk 1 above)
Comment (sbellem) [inline docker-compose.yml:7]: If one wishes to modify the docs, and/or build the docs, and have the changes synced on the host and in the container, yes. We could probably simply mount the whole thing:

``` yaml
- .:/usr/src/app
```

File: docker-compose.yml (hunk 2 above)
Comment (sbellem) [inline docker-compose.yml:18]: yes, for the docstrings, which are used in some parts of the docs

File: docs/source/api.rst (hunk 3 above)
Comment (sbellem) [inline docs/source/api.rst:11]: It is in the `docstrings` already. https://github.com/sbellem/pyspool/blob/fc995ae8f6b8f82ec7a13b2387bef804eb505550/spool/spool.py#L24-L49

:smile:

**Comment by sbellem** [comment]
@SohKai I thought of waiting to remove the docs from the `README` until everything is ported. At the moment `File`, `Wallet`, `Spoolverb`, `BlockchainSpider`, and `Ownership` have  yet to be documented (in sphinx-based docs).

**Pull request merged**